Tools/scripts/summarize_stats.py

```diff
@@ -415,12 +457,21 @@ def get_histogram(self, prefix: str) -> list[tuple[int, int]]:
     def get_rare_events(self) -> list[tuple[str, int]]:
         prefix = "Rare event "
         return [
-            (key[len(prefix) + 1:-1].replace("_", " "), val)
+            (key[len(prefix) + 1 : -1].replace("_", " "), val)
             for key, val in self._data.items()
             if key.startswith(prefix)
         ]
 
 
+class Doc:
```

Comment:
It's a bit odd to see this class defined *after* it is used.

Reply:
I moved these markdown-rendering classes earlier in the file.